en/docs/learn/password-recovery-via-user-preferred-notification-channel.md

```diff
@@ -0,0 +1,597 @@
+            </ul> 
+            <div class="admonition note">
+                <p class="admonition-title">note</p>
+                <p>
+                    If you have configured Username Recovery Via User Preferred Notification 
+                    Channel, the above field might be already configured.
+                </p>
+            </div>   
+          </td>
+       </tr>
+       <tr>
+          <td>resend_code_validity</td>
+          <td>
+            <ul>
+                <li>Validity period of the recovery code given after initiating password recovery.
+                <li>Unit : <code>minutes</code>
+                <li>Default value : <code>1 minute</code>
+            </ul>
+          </td>
+       </tr>
+       <tr>
+         <td>sms_otp_validity</td>
+         <td>
+           <ul>
+                <li>Validity period of SMS OTP when the selected channel is <code>SMS</code>
+                <li>Unit : <code>minutes</code>
+                <li>Default value : <code>1 minute</code>
+           </ul>
+         </td>
+      </tr>
+    </table>
+          
+3.  Add an event publisher to `<IS_HOME/repository/deployment/server/eventpublishers`. For this 
+sample, `http output adapter` is used. Following is a sample publisher to call a REST Service 
+to send confirmation codes.
+
+    ??? info "Sample Event Publisher"
+        ```
+        <?xml version="1.0" encoding="UTF-8"?>
+        <eventPublisher name="HTTPOutputEventAdapter" processing="enable"
+            statistics="disable" trace="disable" xmlns="http://wso2.org/carbon/eventpublisher">
+            <from streamName="id_gov_sms_notify_stream" version="1.0.0"/>
+            <mapping customMapping="enable" type="json">
+                <inline>{"api_key"="4c9374",
+                    "api_secret"="FtqyPE93",
+                    "from"="NEXMO",
+                    "to"={{mobile}},
+                    "text"={{body}}
+                    }</inline>
+            </mapping>
+            <to eventAdapterType="http">
+                <property name="http.client.method">httpPost</property>
+                <property name="http.url">https://rest.nexmo.com/sms/json</property>
+            </to>
+        </eventPublisher>
+        ``` 
+        
+        !!! note
+            This publisher uses NEXMO as the SMS REST service provider. For more information 
+            on writing a custom http event publisher, see [HTTP Event Publisher](https://docs.wso2.com
+            /display/DAS300/HTTP+Event+Publisher).  
+            
+4.  [Start WSO2 IS](../../setup/running-the-product#starting-the-server)
+    and [log in to the management console](../../setup/running-the-product#accessing-the-management-console)
+    : ` https://<IS_HOST>:<IS_PORT>/carbon `.  
+    **NOTE:** If your IS is already running, make sure to restart to apply above configurations. 
+    
+5.  Navigate to `Main -> Identity Providers -> Resident -> Account Management Policies` section.
```

Comment:
**Main -> Identity Providers -> Resident -> Account Management Policies** should be in bold and not monospace. Please apply to all applicable places.